.gitignore

```diff
@@ -181,3 +181,5 @@ Installers/
 
 CodeGraphData/
 /Rubberduck.Deployment/Properties/launchSettings.json
+/Rubberduck.Deployment/Rubberduck.API.idl
+/Rubberduck.Deployment/Rubberduck.idl
```

Comment:
why not just `**/*.idl`?

Reply:
worst case, those can be whitelisted... on the other hand it doesn't make all that much of a difference :)

Reply:
We'd need a static .idl if we ever decided to put the COM collector under test, no?

Reply:
Indeed though they'd be in the `RubberduckTests` project, not here. But I think that's good enough reason to not wildcard them.